languages.json

```diff
@@ -1195,8 +1195,8 @@
     },
     "Vue": {
       "name": "Vue",
-      "multi_line_comments": [["<!--", "-->"]],
-      "quotes": [["\\\"", "\\\""], ["'", "'"]],
+      "line_comment": ["//"],
+      "multi_line_comments": [["<!--", "-->"], ["/*", "*/"],
```

Comment:
```suggestion
      "multi_line_comments": [["<!--", "-->"], ["/*", "*/"]],
```

Reply:
Yeah, forgot the closing `]` thanks :+1: